web/src/pages/index.tsx

```diff
@@ -75,9 +75,6 @@ const Home = (props: Props): ReactElement => {
     };
   }
   useEffect(() => {
-    const landingPageAlternateUrl = process.env.ALTERNATE_LANDING_PAGE_URL || ROUTES.landing;
```

Comment:
[Pebble] I think we should remove the environment variables from the .env-template. I think we can also remove it from bit warden, and the environment variables that are saved in circles ci

Reply:
yep, made a note of it above in the pr comment.

Reply:
The changes to .env-template should be part of this PR

Reply:
They are not there by default in .env-template file.